File: quadratic.py
from math import sqrt


def quadratic(a, b, c):
    d = (b ** 2) - (4 * a * c)
    sol1 = (-b - sqrt(d)) / (2 * a)
    sol2 = (-b + sqrt(d)) / (2 * a)
    return (sol1, sol2)


print(quadratic(1, 7, -4))

File: src/quadratic.cpp
#include <iostream>
#include <math.h>
// #include <boost/python.hpp>
#include "quadratic.h"
__declspec(dllexport) double quadratic(double a, double b, double c) {
  double inside = pow(b, 2) - (4 * a * c);
  double sol1 = (-b + sqrt(inside)) / (2 * a);
  return sol1;
}